asciidoctorj-core/src/test/java/org/asciidoctor/WhenAsciiDocIsRenderedToDocument.java

```diff
@@ -235,7 +235,7 @@ public void should_be_able_to_read_asset() throws FileNotFoundException {
         Document document = asciidoctor.load(DOCUMENT, options);
         File inputFile = classpath.getResource("rendersample.asciidoc");
         String content = document.readAsset(inputFile.getAbsolutePath(), new HashMap<Object, Object>());
-        assertThat(content, is(IOUtils.readFull(new FileReader(inputFile))));
+        assertThat(content, is(IOUtils.readFull(new FileReader(inputFile)).replaceAll("\r","\\")));
```

Comment:
Upgraded to last version of JDK (131) and error banished :man_shrugging: I'll rollback the change.